quickstart/integration-test/playwright.config.ts

```diff
@@ -27,23 +26,43 @@ export default defineConfig({
   /* Opt out of parallel tests on CI. */
   workers: process.env.CI ? 1 : undefined,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: 'list',
+  reporter: [
+    ['list'],
+    ['html', { open: 'never' }]
+  ],
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
   use: {
     /* Base URL to use in actions like `await page.goto('/')`. */
     // baseURL: 'http://127.0.0.1:3000',
 
+    actionTimeout: 20_000,
+    // default timeout for navigation/waitForNavigation
+    navigationTimeout: 30_000,
+
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: 'on-first-retry',
   },
 
   /* Configure projects for major browsers */
   projects: [
+    // {
+    //   name: 'chromium',
+    //   use: { ...devices['Desktop Chrome'] },
+    // },
+
     {
-      name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
+      name: 'login',
+      testMatch: '**/login.spec.ts',
+      // use: { ...devices['Desktop Chrome'] },
     },
 
+    {
+      name: 'workflow',
+      testMatch: '**/workflow.spec.ts',
+      use: { ...devices['Desktop Chrome'] },
```

Comment:
FWIW: we're running the CN frontend integration test using firefox only. So far this has served us well as a stand-in for wide browser compatibility.

Consider switching to Firefox here to protect from unknown unknowns wrt browser compatibility of chrome to other browsers.